neo/IO/Helper.cs

```diff
@@ -92,17 +92,16 @@ public static byte[] ReadBytesWithGrouping(this BinaryReader reader)
         {
             using (MemoryStream ms = new MemoryStream())
             {
-                int padding = 0;
+                int count;
                 do
                 {
                     byte[] group = reader.ReadBytes(GroupingSizeInBytes);
```

Comment:
We should reuse this buffer
```
byte[] group=new byte[GroupSizeInBytes];
do{
read.Read(group,0,GroupSizeInBytes);

```

Reply:
```c#
read.Read(group,0,GroupSizeInBytes);
```

This may read data less than `GroupSizeInBytes` and use an extra loop, which is very troublesome.